containers/fluentd/Dockerfile

```diff
@@ -18,3 +18,6 @@ FROM k8s.gcr.io/fluentd-elasticsearch:v2.0.4
 
 # Install fluentd-s3 plugin
 RUN gem install -N fluent-plugin-s3 -v '~>1.1.4'
```

Comment:
Maybe we should add `--no-document` to `fluent-plugin-s3` too?

Reply:
done in 477a07a